05-dijkstras/06-dijkstras-vs-negative-graphs.md

```diff
@@ -0,0 +1,127 @@
+# Optional: Dijkstra vs. Negative Weighted Graphs
+
+### !callout
+
+## Optional Lesson
+
+This lesson is optional because it is not necessary to learn Dijkstra's Algorithm. However, it is does lend a handing in understanding the algorithm.
```

Comment:
```suggestion
This lesson is optional because it is not necessary to learn Dijkstra's Algorithm. However, it does lend a hand in understanding the algorithm.
```